package.json

```diff
@@ -22,7 +22,10 @@
     "build:scripts": "vite build",
     "build:types": "tsc --project tsconfig.build.json --emitDeclarationOnly",
     "lint": "eslint --ext .tx,.ts .",
-    "version:get": "node -p \"require('./package.json').version\""
+    "version:get": "node -p \"require('./package.json').version\"",
+    "test:vitest": "TZ='UTC' vitest run",
+    "watch:test": "run-p watch:test:*",
+    "watch:test:vitest": "vitest watch"
```

Comment:
These three you copied from the dashboard I think, but we are moving towards simpler scripts. Replace them for widget-like scripts:

```json
{
  "test": "vitest run",
  "test:watch": "vitest"
}
```